js/interactive-guides/modules/content-manager.js

```diff
@@ -450,6 +450,14 @@ var contentManager = (function() {
       return stepInstruction;
     };
 
+    var checkIfInstructionsForStep = function(stepName){
+        if(!stepName){
+            stepName = stepContent.getCurrentStepName();
+        }
+        var stepInstruction = __instructions[stepName];
```

Comment:
The codes from lines 454 thru 457 could be replaced by a call to __getStepInstruction.

Reply:
Done